Fixes #29 - added option to set paper size in config file

djmgit opened the pull request:
Fixes issue #29, added option for paper size. Now users can
specify paper size in config file.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

niccokunzmann commented on badges/8BadgesOnA3.svg:
```diff
@@ -10,8 +10,8 @@
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   width="297mm"
-   height="420mm"
+   width="paper_width"
```
Is the file still editable using inkscape? Changing the size will not change the positions of the elements. Could you check?

djmgit replied:
I checking it, will update PR if necessary  ASAP

djmgit commented:
@jajodiaraghav @niccokunzmann I have added option to set paper size in config file. Paper the given paper size will be mapped to actual dimensions in python script. For now I have added support for A4 and A3. We can make it more flexible by allowing the user to specify custom dimensions in the cconfig file as well. If it sounds good then I would love to iterate on this in subsequent PRs.

niccokunzmann commented:
@djmgit okay, so I will merge it.
